Sources/CollectionViewDriver.swift

```diff
@@ -203,16 +203,11 @@ public class CollectionViewDriver: NSObject, UICollectionViewDataSource, UIColle
     }
 
     public func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
-        var cell: UICollectionViewCell
-
-        if let cellViewModel = self.collectionViewModel?[indexPath] {
-            cell = collectionView.dequeueReusableCell(withReuseIdentifier: cellViewModel.cellIdentifier, for: indexPath)
-            cellViewModel.applyViewModelToCell(cell)
-            cell.accessibilityIdentifier = cellViewModel.accessibilityFormat.accessibilityIdentifierForIndexPath(indexPath)
-        } else {
-            cell = UICollectionViewCell()
-        }
+        guard let cellViewModel = self.collectionViewModel?[indexPath] else { return UICollectionViewCell() }
```

Comment:
I think @jessesquires  was planning on dropping this escape hatch `return UICollectionViewCell()`, in favor of always returning the dequeued cell?

Reply:
Ah yeah can do!